packages/rolldown/tests/fixtures/function/comments/preserve/output.snap

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Finally, the shotsnap for these three test cases seems unexpected. There may be an issue with `oxc_codegen` regarding comments, and I'll open an issue to track it later.